bleak/backends/bluezdbus/client.py

```diff
@@ -377,9 +377,12 @@ def _cleanup_all(self) -> None:
             self.services = BleakGATTServiceCollection()
             self._services_resolved = False
 
-    async def disconnect(self) -> bool:
+    async def disconnect(self, **kwargs) -> bool:
         """Disconnect from the specified GATT server.
 
+        Keyword Args:
+            timeout (float): Defaults to 10.0.
```

Comment:
Perhaps specify the unit as seconds; so "timeout (float): Defaults to 10.0 seconds."